pkg/git/git.go

```diff
@@ -23,14 +23,27 @@ import (
 	"github.com/codefresh-io/cli-v2/pkg/log"
 )
 
+type TokenTypes string
```

Comment:
Generally I think it's better design to leave the token types out of this package and instead just send in the required scopes from outside. A `git` package should not care about `runtime` vs `personal` token.
But you can leave it.